fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -646,7 +791,8 @@ public <T extends Resource> SingleResourceResult<T> update(FHIRPersistenceContex
         final String METHODNAME = "update";
         log.entering(CLASSNAME, METHODNAME);
 
-        try (Connection connection = openConnection()) {
+        try (Connection connection = openConnection();
+                MetricHandle createMetric = FHIRRequestContext.get().getMetricHandle(FHIRPersistenceJDBCMetric.M_JDBC_UPDATE.name())) {
```

Comment:
all FHIRPersistence methods use the same var name "createMetric".  wasn't sure if its from copy-paste or if thats the name you want.  the variable isn't actually used anywhere, so maybe a short name like `m` would suffice?  i think i saw that var name used for these MetricHandle instances elsewhere...

Reply:
Yeah, this is definitely copy-paste. I can tweak the name of the metric handle variable because it doesn't get within the block and so doesn't really matter what it's called.